Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
```

Comment:
I prefer to write the variables with the same name of the clases but in lower case, for example in this case: perfection. I under stand better the code in that form.